processor/probabilisticsamplerprocessor/tracesprocessor_test.go

```diff
@@ -75,7 +86,7 @@ func Test_tracesamplerprocessor_SamplingPercentageRange(t *testing.T) {
 			},
 			numBatches:        1e5,
 			numTracesPerBatch: 2,
-			acceptableDelta:   0.01,
+			acceptableDelta:   0.02,
```

Comment:
I wonder what's the story behind this change in expectations.

Reply:
I reverted these changes.
I had added `cfg.HashSeed = defaultHashSeed` which changes the results sligjhtly--removed it.
Added this comment:
```
	// Note on hard-coded "acceptableDelta" values below.  This
	// test uses a global random number generator: it is sensitive to
	// the seeding of the RNG and it is sensitive to the default
	// hash seed that is used.  This test depends on HashSeed==0.
```